zoo-css-challenge/style.css

```
@@ -188,8 +237,33 @@ footer {
   color: #ff7f11;
 }
 
+/* 7) Membership*/
+.membership p{
+  max-width: 700px;
+  margin-left: 26%;
```

```suggestion
  margin: auto;
```
We can use `margin:auto` which centres the element. `auto` distributes the space left and right equally